common/app/services/player-channel.ts

```diff
@@ -214,6 +224,27 @@ export default class PlayerChannel {
                         );
                     }
                     break;
+                case 'requestSubtitles':
```

Comment:
I don't think this "request subtitles" protocol is necessary as long as `SubtitlePlayer` is publishing subtitle updates, and `SubtitleModel` already carries the coloring information.